frontend/src/redux/store.js

```diff
@@ -0,0 +1,9 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
```

Comment:
don't create a redux folder. 
we are separating our app based on features. everything menu-related should go to the Menu folder, actions, reducers, etc. 
the store can go to the src folder.